docs/root/configuration/http_filters/wasm_filter.rst

```diff
@@ -0,0 +1,1184 @@
+.. _config_http_filters_wasm:
```

Comment:
This documentation should really be part of the C++ SDK, not Envoy proper.

Reply:
Where is c++ sdk? First time heard about his.

Reply:
`api/wasm/cpp`

Reply:
I created a doc/ folder under api/wasm/cpp and move this doc to there.